services/_shared/personas/code_reviewer/claim_check.py

```
@@ -0,0 +1,374 @@
+def _deep_mismatch_message(claimed: Bound, actual: Bound) -> str:
+    code_op = ">" if actual == "exclusive" else ">="
+    claim_op = ">" if claimed == "exclusive" else ">="
+    return (
+        f"Comment/docs describe deep-diff escalation as {claimed} "
+        f"(`added {claim_op} threshold`), but code uses {actual} "
+        f"(`added {code_op} threshold`). Doc/code claim drift - align the "
+        f"bound language with `decide_deep_escalation`."
+    )
+
+
+def _finding_vs_facts(
```

Ruff (0.15.21)

[warning] Too many return statements (8 > 6)

(PLR0911)